examples/src/main/scala/org/apache/spark/examples/mllib/AbstractParams.scala

```diff
@@ -38,7 +38,7 @@ abstract class AbstractParams[T: TypeTag] {
    */
   override def toString: String = {
     val tpe = tag.tpe
-    val allAccessors = tpe.declarations.collect {
+    val allAccessors = tpe.decls.collect {
```

Comment:
Hm, not sure why... somehow remember that I couldn't figure out how to suppress it. There are a few like this that can be suppressed, yes.